attributes/default.rb

```diff
@@ -37,6 +37,14 @@
   'crtfile' => 'self-signed.example.com.crt'
 }
 
+default['splunk']['clustering'] = {
+  'enable' => false,
+  'mode' => 'master', # master|slave|searchhead
```

Comment:
Is `master`/`slave` terminology from Splunk? I'd rather not have the negative connotation of master/slave relationships and prefer a primary/secondary or leader/follower kind of terms.